blog/03-14-2023-Build-Your-Own-Full-Stack-NFT-A.I-Art-Minter-On-The-Celo-Blockchain/Build-Your-Own-Full-Stack-NFT-A.I-Art-Minter-On-The-Celo-Blockchain.md

```diff
@@ -0,0 +1,671 @@
+---
+title: Build Your Own Full Stack NFT A.I Art Minter On The Celo Blockchain
+description: In this tutorial, we will build a full stack nft digital art minter that will generate our art with artificial intelligence
+authors:
+  - name: ✍️ Jonathan Iheme
+url: https://github.com/4undRaiser
+image_url: https://avatars.githubusercontent.com/u/87926451?s=96&v=4
+tags: ['celosage', 'solidity', 'celo', 'nft', 'react', 'intermediate']
+hide_table_of_contents: true
+slug: "/tutorials/build-your-own-full-stack-nft-a.i-art-minter-on-the-celo -blockchain"
+---
+
+![header](../../src/data-tutorials/showcase/intermediate/build_your_own_full_stack_nft_a.i_art_minter_on_the_celo _blockchain.png)
```

Comment:
This was creating issues , but I changed it this time make sure it does not happen in future